src/pkg/repository/manager.go

```diff
@@ -65,6 +67,18 @@ func (m *manager) Get(ctx context.Context, id int64) (*models.RepoRecord, error)
 	return m.dao.Get(ctx, id)
 }
 
+func (m *manager) GetByName(ctx context.Context, name string) (repository *models.RepoRecord, err error) {
+	_, repositories, err := m.List(ctx, &q.Query{
+		Keywords: map[string]interface{}{
+			"Name": name,
```

Comment:
Actually the key of the keywords should be aligned with the property name in the JSON model.

Reply:
lower case : "name"?

Reply:
No json annotation defined for the artifact model used by the controller. What users can see is only the property name of the model. The convert is handled by beego orm